pkg/proxy/conntrackcleaner/cleaner.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conntrackcleaner
```

Comment:
Possibly this should just go into the existing `pkg/util/conntrack`. Or else `pkg/util/conntrack` should be moved into `pkg/proxy`... This all kind of depends on what we end up deciding with staging the proxy sources...

Reply:
@danwinship  yeah, I initially thought about putting this in `pkg/util/conntrack` only, but `pkg/util/conntrack` only deals with IPs and Ports, it isn't aware of kube-proxy/business logic and since it is part of  `pkg/util` tree I avoided adding any `pkg/proxy` dependency there.
Will move `pkg/util/conntrack` to `pkg/proxy`.

Reply:
@danwinship though I'm not sure if some other pkg depends on `pkg/util/conntrack`